arrow/test/test_arrow.py

```diff
@@ -162,59 +160,55 @@ def test_compare_runtime():
     amplify = 100
 
     reference = GillespieReference(stoichiometric_matrix, rates)
-    reference_start = time.time()
+    reference_start = seconds_since_epoch()
```

Comment:
Nice, love `seconds_since_epoch`. What is the benefit here? Is this more accurate somehow?

Reply:
So the problem (pointed out by PyCharm's inspection) is that there are functions and methods with the local variable `time` (a return field from `obsidian.evolve()`) that shadows the imported the module `time` to get the function `time`. This is confusing, at risk of breaking, and at risk of confusing people in a debugger.

The solution is to either rename those local variables (they're copied from an `evolve_state` field named `time`), or rename the imported module, or import/rename the function `time.time()`. The latter has a very poor name. It gives no clue that it's a getter, the time format, scale, resolution, or that it changes when adjusting the system clock. So it wins the rename-me choice.

This PR renamed some more shadowed variables. Unfortunately variables declared in `if __name__ == '__main__'` code are global. That's one reason to move the bulk of such code into a `main()` function.